Api/WebScrap.API.Tests/Features/DeepChildLocationTests.cs

```diff
@@ -53,10 +53,10 @@ public void Test3()
     {
         var css ="div.container b";
         var html = """
-        <main class="container">
+        <div class="container">
             <p> Lorem <b> Ipsum </b> </p>
             <b> Clatu <b> Barata <b> Nictu </b> </b> </b>
-        </main>
+        </div>
```

Comment:
I see div has a direct child here.
Whats the point?
Suggest: wrap Clatu words with div or p or whatever.